Bio/SeqUtils/CodonUsage.py

```diff
@@ -184,4 +177,4 @@ def print_index(self):
         This just gives the index when the objects is printed.
         """
         for i in sorted(self.index):
-            print("%s\t%.3f" % (i, self.index[i]))
+            print("%s\t%s" % (i, self.index[i] is None and 'None' or "{:.3f}".format(self.index[i])))
```

Comment:
Style query: Can you change ``'None'`` to ``"None"`` (or run the command line tool ``black``).

Functionality: Is the idea here to to show the string None where the index is None?

Suggestion: Can you add a doctest to the ``print_index`` docstring?

Reply:
Style query: Thank you! implemented.
Functionality: yes.
Suggestion: ha, I am not very familiar with doctest. Just added a test following doctest help page. Wonder if that's what you mean.

Reply:
the changes to `Bio/Cluster/__init__.py` is intentional because the docstring for function `kmedoids` is just a copy-paste of that for `kcluster` and this does not make sense. Would love to get help on git. Thank you!